src/components/Player.vue

```diff
@@ -230,6 +239,16 @@ export default {
     handleEmojis: text => text.replace(/:([^: ]+?):/g, "").replace(/ •/g, "\n•")
   },
   methods: {
+    changePronoun() {
+      if (!this.session.isSpectator || this.player.id !== this.session.playerId)
+        return;
+      const pronoun = prompt("Player preffered pronouns", this.player.pronoun);
```

Comment:
I'd add a rather strict length limit here, maybe 20 chars?

Reply:
I have added a max width of 250px and overflow hidden to the pronouns modal, so anything too long just wont be displayed.